packages/react-html/src/server/components/Html.tsx

```diff
@@ -33,7 +33,7 @@ export default function Html({
   headMarkup = null,
   bodyMarkup = null,
 }: Props) {
-  const markup = renderToString(children);
+  const markup = renderToString(<>{children}</>);
```

Comment:
this was necessary, because `renderToString` expects a single `ReactElement`, but the `Html` component allows any `ReactNode` including multiple or none.